platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -148,7 +148,8 @@
         'thermal': {
             "capability": {
                 "comex_amb": False
-            }
+            },
```

Comment:
a file that is updated requires some change in the copyright. please review and make sure it is updated